sklearn/metrics/cluster/_unsupervised.py

```diff
@@ -59,19 +59,19 @@ def silhouette_score(X, labels, *, metric='euclidean', sample_size=None,
 
     Parameters
     ----------
-    X : array [n_samples_a, n_samples_a] if metric == "precomputed", or, \
-             [n_samples_a, n_features] otherwise
-        Array of pairwise distances between samples, or a feature array.
+    X : array-like of shape (n_samples_a, n_samples_a) if metric == \
+            "precomputed", (n_samples_a, n_features) otherwise
```

Comment:
I would prefer:

```suggestion
    X : array-like of shape (n_samples_a, n_samples_a) if metric == \
            "precomputed" or (n_samples_a, n_features) otherwise
```

instead.

Reply:
OK, this will make it more readable on the HTML. Thanks

Reply:
@OlehKSS Can you accept the suggestions?

Reply:
Sure